tslint.json

```diff
@@ -0,0 +1,25 @@
+{
+  "extends": "tslint:recommended",
+  "rules": {
+    "quotemark": [
+      "single",
+      "avoid-escape"
+    ],
+    "no-var-requires": false,
+    "arrow-parens": false,
+    "only-arrow-functions": [
+      false
+    ],
+    "member-access": false,
+    "member-ordering": [
+      "field-first"
+    ],
+    // TODO: these should be activated at one point
```

Comment:
It works though as is normally done like this, see the tslint.json from the maintainers of tslint: https://github.com/palantir/tslint/blob/master/tslint.json